contrib/completion/powershell/posh-docker.psm1

```diff
@@ -0,0 +1,179 @@
+# Powershell completion for docker
```

Comment:
if we're going to copy it from https://github.com/samneirinck/posh-docker don't we need to follow his LICENSE and copy the copyright notice?